### ISSUE TITLE
Re-train model on training set

### ISSUE THREAD

**Issue by amitSA** [catherinelee274/fraud_detection#31]
We should try to train the model on the full 200,000,000 training set on Kaggle Notebook.
  - They have an option to train with GPUs

We won't be able to equalize the set (because we're using the full training set), but with the full 200 million clicks it might not matter.  The cons with an equalized training set is that `click_time` attributes cannot be used.

Although we might be able to compromise and create an equalized training set that is larger than 100,000 clicks.

**Comment by amitSA** [comment]
Todo

- [ ] Convert click_time into pandas timestamp values
- [ ] Create 'click_time_secs' through pandas because it's shorter code and at least 10X faster than the current way

**Comment by amitSA** [comment]
I tried to re-train on the first 10 million rows in the training set, but got 57% accuracy on the test set.  
This most likely is because is_attribute's value distributions are skewed 99% on '0' in the training set while 50% in test set.   Hence the need for equalizing training sets.

Maybe if we fine turned our equalized set to be bigger but preserve click count ratios accros the features, we'll get better accuracy
